### DISCUSSION TITLE
Can I convert QSM outcomes into DICOM format?

### DISCUSSION THREAD

**Discussion by kaiyuliu123** [QSMxT/QSMxT#106] [Q&A] [answered]
Hello guys, I want to know if it is possible to convert QSM outcomes(.nii.gz) into DICOM files after I've gone through the QSMxT pepline. I tried to load my QSM outcome with nii.gz format in 3Dslier and export it to dicom but failed. The dicom directory generated was filled with blank images. Could someone offer some advice regarding this issue since dcm files are more practical in clinics and is sometimes the only format alloowed in steretaxic surgery planning systems such as Brainlab.

**Comment by astewartau** [comment, marked as answer]
Hi @kaiyuliu123, thanks for your question - this is now possible in QSMxT v6.4.0!

See the [release notes](https://github.com/QSMxT/QSMxT/releases/tag/v6.4.0) and install using the [usual instructions](https://qsmxt.github.io/QSMxT/installation). 

Please let me know if you have any issues with this feature. 😊
